time.go

```diff
@@ -15,16 +15,24 @@ var timeFormats = []string{
 	"02/01/2006",
 	"02/01/2006 15:04",
 	"02/01/2006 15:04:05",
+	// time zone formats
 	"2006-01-02T15:04-07:00",
 	"2006-01-02T15:04:05-07:00",
+	"2006-01-02T15:04:05Z07:00",
+	"2006-01-02 15:04:05Z07:00",
+	"2006-01-02T15:04:05Z0700",
+	"2006-01-02 15:04:05Z0700",
+	"2006-01-02T15:04:05Z07",
+	"2006-01-02 15:04:05Z07",
+	"2006-01-02 15:04:05 MST",
 }
 
 // ErrCouldNotConvertToTime is returns when a string is not a reconizable time format
 var ErrCouldNotConvertToTime = errors.New("Could not convert string to time")
 
 func tryStrToDate(s string) (time.Time, error) {
 	for _, f := range timeFormats {
-		t, err := time.ParseInLocation(f, s, time.Local)
+		t, err := time.Parse(f, s)
```

Comment:
We need to keep the previous version here, otherwise this would be a breaking change.

From the [time.Parse](https://golang.org/pkg/time/#Parse) documentation:

> In the absence of a time zone indicator, Parse returns a time in UTC. 

From the [time.ParseInLocation](https://golang.org/pkg/time/#ParseInLocation) documentation:

> In the absence of time zone information, Parse interprets a time as UTC; ParseInLocation interprets the time as in the given location.

I'll create an issue to remember to add an additional option in v3 to choose a given time zone for testfixtures to use. For now, it must use `time.Location`.

Reply:
OK gotcha. If you want I can go ahead and add the option for using `time.Parse`. Then default behavior can remain `time.ParseInLocation`.

Reply:
Let's keep it as is or v2.

I already started working on v3 and then I'll add that time zone option.